### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by metatrevor**
Here are the changes made so far.

### CODE REVIEW

File: django_pesapal/app.py [hunk 1]
```diff
@@ -114,3 +105,60 @@ def get_payment_url(request=None, **kwargs):
...
+
+    return response_data
+
+def get_transaction_model():
+    from django.db.models import get_model
+
+    try:
+        app_label, model_name = settings.PESAPAL_TRANSACTION_MODEL.split('.')
+    except ValueError:
+        raise ImproperlyConfigured("PESAPAL_TRANSACTION_MODEL must be of the form 'app_label.model_name'")
```
Comment (odero) [inline django_pesapal/app.py:159]: This assumes only 1 level of indirection. What if my app is structured as `module.something.app.model`?

File: django_pesapal/views.py [hunk 2]
```diff
@@ -21,10 +22,56 @@ def get(self, request, *args, **kwargs):
...
         return super(TransactionCompletedView, self).get(request, *args, **kwargs)
+
+    def get_redirect_url(self, *args, **kwargs):
+        url = reverse_lazy(conf.PESAPAL_TRANSACTION_DEFAULT_REDIRECT_URL,
+                           kwargs={'transaction_id': self.transaction_id, 'merchant_reference': self.merchant_reference})
+        return url
+
+
+class IpnResponseHandlerView(View):
+    permanent = False
```
Comment (odero) [inline django_pesapal/views.py:47]: What is this `permanent = False` for?

File: django_pesapal/app.py (hunk 1 above)
Comment (metatrevor) [inline django_pesapal/app.py:159]: Noted

File: django_pesapal/views.py (hunk 2 above)
Comment (metatrevor) [inline django_pesapal/views.py:47]: That's definitely misplaced :D

**Pull request closed without merging**